### ISSUE TITLE
comments_popup_link defaults for accessibility

### ISSUE THREAD

**Issue by samikeijonen** [Automattic/_s#907]
I was wondering how `_s` uses `comments_popup_link` function:

https://github.com/Automattic/_s/blob/master/inc/template-tags.php#L64

```
comments_popup_link( esc_html__( 'Leave a comment', '_s' ), esc_html__( '1 Comment', '_s' ), esc_html__( '% Comments', '_s' ) );
```

Core take care of this a little better by default what it comes for accessibility. For example Core uses for `1 Comment` string like this:

```
sprintf( __( '1 Comment<span class="screen-reader-text"> on %s</span>' ), $title )
```

Would it be better to use defaults for this one?

**Comment by iamdmitrymayorov** [comment]
I believe it is done this way because it is generally not a good thing to let translators handle the markup. But I agree that adding the .screen-reader-text class would benefit the accessibility.

**Comment by samikeijonen** [comment]
I translate Core and it's not a problem:)

Actually this Core defaults are better for two reasons:

1. Better accessibility.
2. Fewer strings to translate in your theme.

**Comment by iamdmitrymayorov** [comment]
Agree, but what I'm saying is, we can still go the accessibility road, without markup in translatable strings:
```
sprintf( __( '1 Comment on %s' ), '<span class="screen-reader-text">' . $title . '</span>' )
```

**Comment by samikeijonen** [comment]
That would look weird on front end for non screen reader users. `1 Comment on` and then nothing.

**Comment by iamdmitrymayorov** [comment]
Silly me, didn't think of that :)

**Comment by obenland** [comment]
We could probably change the first argument to `'Leave a Comment<span class="screen-reader-text"> on %s</span>'` and lose the rest.

**Comment by samikeijonen** [comment]
That would be step forward. I'd even leave Core handle the first argument, but that's just me:)

**Comment by obenland** [comment]
I thought about that too, but it *is* more encouraging than the default wording.

**Comment by samikeijonen** [comment]
Do you want me to create pull request or what are we waiting for:)

**Comment by karmatosed** [comment]
@samikeijonen if you can do a PR that would be great. I'd suggest what @obenland did to have more encouraging wording.

**Comment by samikeijonen** [comment]
Ok, I'll send a PR.

**Comment by samikeijonen** [comment]
I used `wp_kses`, hope that's ok.

**Comment by sixhours** [comment]
Looks like this was merged in #915, closing.
